source/fundamentals/builders/aggregates.txt

```diff
@@ -50,6 +48,32 @@ aggregation as a list:
    :language: java
    :dedent:
 
+Documents
+---------
```

Comment:
Q: Is this new feature significant enough to be at the top of the page, even before match and project?

Reply:
Good question. My reasoning was that when this stage it's used, it will be the first in the pipeline. But I'm happy to move it down if you don't find that reason compelling.

Reply:
I think that match will likely still be the first stage for most users use cases... maybe this can go after project so it's near the top but not first. But up to you!